core/src/main/java/org/lflang/analyses/statespace/Tag.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Class representing a logical time tag, which is a pair that consists of a timestamp (type long)
+ * and a microstep (type long).
+ *
+ * @author{Shaokai Lin <shaokai@berkeley.edu>}
+ */
+package org.lflang.analyses.statespace;
```

Comment:
Javadoc comments should be before the class declaration, not before the package declaration. As it is these comments are dangling and will be ignored by the javadoc engine, including IDEs.

Reply:
Ah, I see. Thanks for the explanation.